packages/documentation-site/patternfly-docs/pages/home.js

```diff
@@ -1,278 +1,317 @@
-                  </Text>
-                </FlexItem>
-              </Flex>
-            </GridItem>
-            <GridItem span={12} lg={11} className="ws-pf-principle">
-              <Flex>
-                <FlexItem>
-                  <img
-                    className="ws-principles-img"
-                    src={scale}
-                    aria-hidden
-                  />
-                  </FlexItem>
-                  <FlexItem>
-                  <Text component={TextVariants.h2}>Make it scalable.</Text>
-                  <Text component={TextVariants.p} className="pf-v5-u-mb-3xl ws-mdx-p">
-                  Helping teams work productively at scale is at the heart of everything we do. Our components are designed for enterprise IT applications used in businesses of all sizes.
-                  </Text>
-                </FlexItem>
-              </Flex>
-            </GridItem>
-            <GridItem span={12} lg={11} className="ws-pf-principle">
-              <Flex>
-                <FlexItem>
-                  <img
-                    className="ws-principles-img"
-                    src={open}
-                    aria-hidden
-                  />
-                  </FlexItem>
-                  <FlexItem>
-                  <Text component={TextVariants.h2}>Make it open.</Text>
-                  <Text component={TextVariants.p} className="pf-v5-u-mb-3xl ws-mdx-p">
-                    We're focused on shaping the future of open source design and development, and we can't do this alone. So we work as an open source community, where everyone is free to share their ideas and contribute their work.
-                  </Text>
-                </FlexItem>
-              </Flex>
-            </GridItem>
-          </Grid>
-        </TextContent>
-        <div className="pf-v5-u-pb-md">
-          <Text
-            className="pf-v5-u-pl-0 pf4-c-link__text ws-mdx-a"
-          >
-            <Link to="/get-started/develop" className="pf-m-white ws-get-started" aria-label="get started developers">
-              Get started for developers <ArrowRightIcon />
-            </Link>
-          </Text>
-        </div>
-        <div className="pf-v5-u-pb-md">
-          <Text
-            className="pf-v5-u-pl-0 pf4-c-link__text ws-mdx-a"
-          >
-            <Link to="/get-started/design" className="pf-m-white ws-get-started" aria-label="get started designers">
-              Get started for designers <ArrowRightIcon />
-            </Link>
-          </Text>
-        </div>
-      </GridItem>
-    </Grid>
-  </PageSection>
+        </FlexItem>
+        <Button
+          isInline
+          size="lg"
+          component="a"
+          href="https://github.com/patternfly"
+          variant="link"
+          aria-label="Patternfly on Github"
+          style={{float: "right"}}
```

Comment:
You shouldn't need this as far as I can tell?

Reply:
Good catch